src/app.ts

```diff
@@ -22,14 +22,18 @@ async function start() {
         storage: "http://localhost:7070",
     };
 
-    // This configures the FrsClient to use a remote Azure Fluid Service instance.
```

Comment:
Let's also keep this comment `// This configures the FrsClient to use a remote Azure Fluid Service instance.`